packages/material-ui/src/Chip/Chip.test.js

```diff
@@ -423,7 +423,7 @@ describe('<Chip />', () => {
           key: ' ',
         };
         wrapper.find('div').simulate('keyDown', spaceKeyDown);
-        assert.strictEqual(preventDefaultSpy.callCount, 2);
```

Comment:
Might make sense to extract the testing-library port from #18271 to see that everything is working.